optional_components/speech_api/app/voice_components.py

```diff
@@ -6,16 +6,15 @@
 
 logger = setup_logger("Whisper ASR")
 
-model = whisper.load_model(PREFERRED_MODEL, download_root=WHISPER_MODEL_DIR)
-logger.info(f"Whisper model '{PREFERRED_MODEL}' loaded successfully.")
-
 
 async def transcribe_audio(file_path: str) -> TranscriptionResponse:
```

Comment:
SO you loading the model every time you transcribe it? Wouldn't that make it slow?

Reply:
i fixed this in the upcoming refactoring of the endpoint PR